java/src/test/java/io/cucumber/gherkin/GherkinDialectsTest.java

```diff
@@ -0,0 +1,17 @@
+package io.cucumber.gherkin;
+
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class GherkinDialectsTest {
+    @Test
+    void each_dialect_has_at_least_one_given_keyword_except_the_star() {
```

Comment:
Nice find. If we test this, this should be a language agnostic test against `gherkin-language.json`. But I'd rather fix this in Cucumber-JVM with a "Generic" `@Step` annotation/method to fall back to if we can't work out the previous GWT. 

Could you please create an issue for that against Cucumber-JVM?